tracker/templates/track.json.j2

```diff
@@ -0,0 +1,32 @@
+{% raw %}{% import "rally.helpers" as rally with context %}{% endraw %}
+{
+  "version": 2,
+  "description": "Tracker-generated track for {{track_name}}",
+  "indices": [{% for index in indices %}
+    {
+      "name": "{{index.name}}",
+      "body": "{{index.filename}}"
+    }{% if not loop.last %},{% endif %}{% endfor %}
+  ],
+  "corpora": [
+    {
+      "name": "{{track_name}}",
+      "base-url": "{{corpora[0].base_url}}",
+      "documents": [{% for corpus in corpora %}
+        {
+          "target-index": "{{corpus.index_name}}",
+          "source-file": "{{corpus.filename}}",
+          "document-count": {{corpus.doc_count}},
+          "compressed-bytes": {{corpus.compressed_bytes}},
+          "uncompressed-bytes": {{corpus.uncompressed_bytes}}
+        }{% if not loop.last %},{% endif %}{% endfor %}
+      ]
+    }
+  ],{% raw %}
+  "operations": [
```

Comment:
IMHO we should use the simplest possible configuration and should define operations inline and only allow one challenge (see also our first example in the [Rally docs](https://esrally.readthedocs.io/en/stable/adding_tracks.html)). This is not as maintainable but also much simpler to get started.

Reply:
Addressed in https://github.com/elastic/rally/pull/875/commits/ff4283398bdce2db2d7eb81158d6f39942347d2f